api/libproject/devstart.rb

```diff
@@ -1136,6 +1136,46 @@ def list_clusters(cmd_name, *args)
   :fn => ->(*args) { list_clusters("list-clusters", *args) }
 })
 
+def create_local_es_index(cmd_name, *args)
+  op = WbOptionsParser.new(cmd_name, args)
+  op.opts.inverse_prob = 10000
```

Comment:
nitpick: actually default to 1 before this goes in.

Reply:
Makes sense – in that case, update the option description to match.

Reply:
Oops, no wonder you asked for the change. Done.